cmd/ncproxy/ncproxy.go

```diff
@@ -599,14 +649,29 @@ func (s *grpcService) GetNetworks(ctx context.Context, req *ncproxygrpc.GetNetwo
 // TTRPC service exposed for use by the shim.
 type ttrpcService struct {
 	containerIDToComputeAgent *computeAgentCache
+	agentStore                *computeAgentStore
```

Comment:
It would be good to add a comment here clarifying what the difference is between a "store" and a "cache". That's distinction is a bit confusing currently.

Reply:
done